x-pack/plugins/observability_solution/infra/public/components/shared/templates/infra_page_template.tsx

```diff
@@ -118,10 +130,22 @@ export const InfraPageTemplate = ({
     />;
   }
 
+  const evaluateNoDataConfig = () => {
```

Comment:
I wonder if this logic should be moved to `getNoDataConfig` function. IMO that's the function responsible for returning the `noDataConfig`

Reply:
We can do it, yes. I’ve tried to modify as little as possible.

Reply:
Changed in https://github.com/elastic/kibana/pull/196387/commits/3ee8a24c10da4a3d4046509c27d99f01a8e0442e

Reply:
Changed in https://github.com/elastic/kibana/pull/196387/commits/3ee8a24c10da4a3d4046509c27d99f01a8e0442e @crespocarlos